code/_onclick/hud/alert.dm

```diff
@@ -86,6 +86,10 @@
 		client.screen -= alert
 	qdel(alert)
 
+// Proc to check for an alert
+/mob/proc/has_alert(category)
+	return !!(alerts[category])
```

Comment:
```suggestion
	return !isnull(alerts[category])
```

I'm assuming `alerts[category]` can't be false, in which case no need to get truthiness.